### PR TITLE
Fix misc docs typos and formatting

### PR DESCRIPTION
**Pull request by edwardleardi**
Some small typo fixes and formatting in various rst files and py docstrings

### CODE REVIEW

File: torch/nn/functional.py [hunk 1]
```diff
@@ -2270,7 +2270,7 @@ def poisson_nll_loss(input, target, log_input=True, full=False, size_average=Non
             is set to ``False``, the losses are instead summed for each minibatch. Ignored
             when reduce is ``False``. Default: ``True``
         eps (float, optional): Small value to avoid evaluation of :math:`\log(0)` when
-            :attr:`log_input`=``False``. Default: 1e-8
+            :attr:`log_input` is ``False``. Default: 1e-8
```
Comment (edwardleardi) [inline torch/nn/functional.py:2273]: Fixes weird formatting in https://pytorch.org/docs/master/generated/torch.nn.parallel.DistributedDataParallel.html#distributeddataparallel for the ** device_ids** parameter

File: torch/nn/parallel/distributed.py [hunk 2]
```diff
@@ -194,23 +194,23 @@ class DistributedDataParallel(Module):
...
-        Using DistributedDataParallel in conjuction with the
+        Using DistributedDataParallel in conjunction with the
         :ref:`distributed-rpc-framework` is experimental and subject to change.
 
     Args:
         module (Module): module to be parallelized
         device_ids (list of int or torch.device): CUDA devices. This should
                    only be provided when the input module resides on a single
-                   CUDA device. For single-device modules, the ``i``th
+                   CUDA device. For single-device modules, the ``i``\ th
```
Comment (edwardleardi) [inline torch/nn/parallel/distributed.py:204]: Fixes weird formatting in https://pytorch.org/docs/master/generated/torch.nn.parallel.DistributedDataParallel.html#distributeddataparallel for the ** device_ids** parameter

File: torch/nn/functional.py (hunk 1 above)
Comment (edwardleardi) [inline torch/nn/functional.py:2273]: Fixes formatting in https://pytorch.org/docs/master/nn.functional.html#poisson-nll-loss for **eps** parameter

**Review by jlin27 — APPROVED** [review]
